src/loader/patch.h

```diff
@@ -9,7 +9,8 @@ typedef enum {
     PATCH_NO_OP,    // patch operation was not performed
 } patch_status_t;
 
-patch_status_t patchLibrary(const char*);
+bool isMusl();
+patch_status_t patchLibrary(const char*, bool);
```

Comment:
```suggestion
patch_status_t patchLibrary(const char *, bool);
```
nit, not related with Your changes but since You change the definition of the function we can introduce the code style inline for it

Reply:
done